frontend/src/pages/HomePage.tsx

```diff
@@ -26,19 +50,16 @@ export const HomePage = () => {
           variant="h6"
           textAlign={'center'}
```

Comment:
A little bonus challenge for you (it's not critical so we can merge the PR without it): on the homepage there's a lot of space at the top of the page which looks a little strange on mobile.
 
![IMG_DAB1F401AD19-1](https://user-images.githubusercontent.com/24561449/210120482-5ce9f38c-b460-4955-8970-514a350a8343.jpeg)